mne/bem.py

```diff
@@ -1063,14 +1063,20 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
     run_subprocess(cmd, env=env, stdout=sys.stdout, stderr=sys.stderr)
 
     if op.isfile(T1_mgz):
-        # XXX : do this with python code
+        new_info = _extract_volume_info(T1_mgz)
+        if new_info is None:
+            warn('nibabel is required to replace the volume info. Volume info'
```

Comment:
This works with older nibabel too as it default to mne function to write the geometry when nibabel is too old.